### PR TITLE
change buttons naming in material-dialog component

### PR DESCRIPTION
**Pull request by vladimirDobrik**

### CODE REVIEW

File: client/src/app/pages/card-editor/card-editor.component.ts [hunk 1]
```diff
@@ -20,8 +19,16 @@ export class CardEditorComponent implements OnInit, OnDestroy {
   public selectedCardId$ = this.cardsFacade.selectedCardId$;
   public checkedSkills$ = this.skillsFacade.checkedSkills$;
   public skillsList$ = this.skillsFacade.allSkills$;
-  public popupTitle = 'Data is not saved.';
-  public popupText = 'The new card is not saved. The entered data will be permanently deleted';
+
+  public popupTextContent: PopupTextContent = {
```
Comment (vharadkou) [inline client/src/app/pages/card-editor/card-editor.component.ts]: this is private property

File: client/src/app/components/card-detail/card-detail.component.ts [hunk 2]
```diff
@@ -20,8 +20,14 @@ export class CardDetailComponent implements OnChanges {
   public skills = new FormControl();
   public skillsList$: Observable<Skill[]> = this.skillsFacade.allSkills$;
 
-  public popupTitle = 'Delete card.';
-  public popupText = 'Card will be permanently deleted. Are you sure?';
+  public popupTextContent: PopupTextContent = {
```
Comment (vharadkou) [inline client/src/app/components/card-detail/card-detail.component.ts]: should be private

File: client/src/app/pages/card-editor/card-editor.component.ts [hunk 3]
```diff
@@ -20,8 +19,16 @@ export class CardEditorComponent implements OnInit, OnDestroy {
...
+    title: 'Data is not saved.',
+    text: 'The new card is not saved. The entered data will be permanently deleted',
+    buttonText: {
+      cancel: 'Cancel',
+      confirm: 'Dismiss changes'
+    }
+  };
+
   public subscription: Subscription;
```
Comment (vharadkou) [inline client/src/app/pages/card-editor/card-editor.component.ts]: should be private

**Comment by lichmanuyk** [comment]
[Change buttons naming in material-dialog component.](https://trello.com/c/HK098AmS/56-change-buttons-naming-in-material-dialog-component)

**Pull request merged**